### DISCUSSION TITLE
Omni-Rig Support Request

### DISCUSSION THREAD

**Discussion by Jeffrey-Miller** [AlexandreRouma/SDRPlusPlus#389] [Ideas]
Hi Alexandre,  My complements to you and all the contributors on a very nice product. I saw a similar request in comments somewhere and I would like to add my vote for a way to have SDR++ follow my transceiver frequency. I use Omni-Rig, but other ways that do not interfere with concurrent JTAlert or WSJT-DX would be fine. Regards, - Jeff AJ6LG

**Comment by AlexandreRouma** [comment]
Hello,
SDR++ already support an open source standard for controlling it remotely called RigCTL.
I'd suggest asking the developers of OmniRig to support that standard.
If you're already using RigCTL to control SDR++ with WSJT-X, you can simply add a second instance of the RigCTL server and configure it on a different port.
